File: In_C_Programming/q6.c
#include <stdio.h>
int count_Max(int arr[], int ln)
{

    int max = arr[0];

    for (int i = 0; i < ln; i++)
    {
        if (arr[i] > max)
        {
            max = arr[i];
        }
    }

    return max;
}

int main()
{

    int size;

    printf("Enter THe Array Length: \n");
    scanf("%d", &size);

    int arr[size];

    printf("Enter The Elements: \n");

    for (int i = 0; i < size; i++)
    {
        scanf("%d", &arr[i]);
    }

    int result = count_Max(arr, size);

    printf("The Max element of the Array is: %d \n", result);

    return 0;
}

File: In_Java_Programming/q6.java
import java.util.Scanner;

public class q6 {

    public static int getMax(int arr[], int size) {

        int max = arr[0];

        for (int i = 0; i < size; i++) {

            if (arr[i] > max) {

                max = arr[i];
            }
        }

        return max;
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter The Array Length: ");
        int ln = sc.nextInt();

        int x[] = new int[ln];

        int result = getMax(x, ln);

        System.out.println("The Max Element is: " + result);

        sc.close();
    }
}
